test-specifications/cryptoService/accountCreateTransaction.md

```diff
@@ -0,0 +1,156 @@
+| 3       | Creates an account and set the auto renew period to 10 days (864,000 seconds)   | key=<VALID_KEY>, autoRenewPeriod=864000  | The account creation fails with an AUTORENEW_DURATION_NOT_IN_RANGE response code from the network. | N                 |
+
+### **Memo:**
+
+- The memo associated with the account (UTF-8 encoding max 100 bytes).
+
+| Test no | Name                                                       | Input                                                                                                                                 | Expected response                                                               | Implemented (Y/N) |
+|---------|------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|-------------------|
+| 1       | Creates an account with a memo                             | key=<VALID_KEY>, memo="testmemo"                                                                                                      | The account creation succeeds and the account's memo equals “testmemo”.         | N                 |
+| 2       | Creates an account with a memo that exceeds 100 characters | key=<VALID_KEY>, memo="this is a really long memo that definitely exceeds 100 characters and it should without a doubt fail the test" | The account creation fails with a MEMO_TOO_LONG response code from the network. | N                 |
+
+### **Max Automatic Token Associations:**
+
+- The maximum number of tokens with which an account can be implicitly associated. Defaults to 0 and up to a maximum value of 1000.
+
+| Test no | Name                                                                       | Input                                          | Expected response                                                                                                                | Implemented (Y/N) |
+|---------|----------------------------------------------------------------------------|------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates an account with a max token association set to maximum value       | key=<VALID_KEY>, maxAutoTokenAssociations=1000 | The account creation succeeds and the account has 1000 automatic token associations.                                             | N                 |
+| 2       | Creates an account with a max token association set over the maximum value | key=<VALID_KEY>, maxAutoTokenAssociations=1001 | The account creation fails with a REQUESTED_NUM_AUTOMATIC_ASSOCIATIONS_EXCEEDS_ASSOCIATION_LIMIT response code from the network. | N                 |
+
+### **Staked ID:**
```

Comment:
Number 6 was removed because `stakedNodeId` is now an `int` instead of a `string`, so it can't be empty. Number 7 was removed by mistake and re-added, thanks for catching that